<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaDiocesis</class>
 <widget class="QDialog" name="dlgNuevaDiocesis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNombre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre latín</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNombreLatin"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="ckArchidiocesis">
       <property name="text">
        <string>Archidiócesis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckExiste">
       <property name="text">
        <string>Existe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Archidiócesis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtArchidiocesis"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarArchi">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Lugar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtLugar"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarLugar">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckBuscado">
     <property name="text">
      <string>Buscado</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtURL"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Parroquias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spParroquias"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>line</zorder>
  <zorder>ckBuscado</zorder>
  <zorder>btOK</zorder>
  <zorder>btCancelar</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
